ta/pkcs11/src/pkcs11_token.c

```diff
@@ -115,6 +115,13 @@ uint32_t entry_ck_slot_list(uint32_t ptypes, TEE_Param *params)
 	return PKCS11_CKR_OK;
 }
 
+static void padd_str(uint8_t *str, size_t size)
```

Comment:
Nit: commit mentions `pad_str` this is `padd_str` with an additional `d`

Reply:
right, thanks. I think `pad_str()` is the preferred label.